src/Symfony/Component/ExpressionLanguage/Parser.php

```diff
@@ -178,21 +178,29 @@ protected function parseConditionalExpression(Node\Node $expr)
     {
         while ($this->stream->current->test(Token::PUNCTUATION_TYPE, '?')) {
             $this->stream->next();
-            if (!$this->stream->current->test(Token::PUNCTUATION_TYPE, ':')) {
+            if ($this->stream->current->test(Token::PUNCTUATION_TYPE, '?')) {
```

Comment:
Won't this way of doing it allow a space between both `?`? Can you please have a look? If yes, then we'll need to patch the lexer.